CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+First off, thank you for considering contributing to Suitest JS API.
+
+When contributing to this repository, please first discuss the change you wish to make via GitHub issue
+with the owners of the repository.
+
+Please note we have a [code of conduct][]. Follow it in all your interactions with the project.
```

Comment:
where is that code? I think that at this stage it may be unnecessary.

Reply:
There is a link to it. It's in the bottom. This is the way GitHub links are usually made - you have [text][] in body and then [text]: http://... in footer.